src/studiodrm-dashjs.js

```diff
@@ -76,27 +77,31 @@
         overrideProtectionKeyController,
         true,
     );
-    player.initialize(document.querySelector("#vuplay-video"), streamURL, true);
+    player.initialize(document.querySelector("#studiodrm-video"), streamURL, true);
     player.attachTTMLRenderingDiv(
         document.querySelector("#ttml-rendering-div"),
     );
 
     // For PlayReady the VUDRM token is attached as a querystring parameter on the license server URL.
     var playReadyLaUrl =
-        "https://playready-license.drm.technology/rightsmanager.asmx?token=" +
-        encodeURIComponent(vudrmToken);
+        "https://playready-license.drm.technology/rightsmanager.asmx";
     // For Widevine set the LaURL and the VUDRM token.
     var widevineLaUrl = "https://widevine-proxy.drm.technology/proxy";
 
-    // Set the protection data. dash.js only supports PlayReady and Widevine but Vualto do support!
+    // TODO: Change lines 97, 103 with the new header name... X-STUDIODRM-TOKEN?
+    // Set the protection data. dash.js only supports PlayReady and Widevine but JW Player do support!
```

Comment:
"dash.js only supports PlayReady and Widevine but JW Player do support!"

what is this meant to say? :)

Reply:
I think just remove the "but...." bit on the end.

Reply:
Actioned.